### ISSUE TITLE
Application Icon

### ISSUE THREAD

**Issue by JellosPorsche** [zaaml/lib#3]
First off, very cool library that solves all of the various non-client area issues with WPF.
Thanks for creating this.

The only thing that seems to be missing is the application icon in the title bar.
Do you plan on adding this to the library?

**Comment by zaaml** [comment]
@JellosPorsche, thank you for your words about the library.

Regarding your question, do you mean the Icon property on the Window object, which is not currently displayed in the themed windows (AppWindow, FloatingWindow) title bar?

**Comment by JellosPorsche** [comment]
Yes, I believe it is the hIcon property.

**Comment by zaaml** [comment]
The issue has been fixed in v0.2.1-alpha.23. The icon size is limited to 24px. There are also new properties for WindowBase class:
`TitleBarHeadContent`, `TitleBarTailContent`. Using these properties you can place custom content in the window title bar. You can place a ToolBarDropDownButton with custom icon and create a window menu:

```xaml
<zm:AppWindow.TitleBarHeadContent>
  <zm:ToolBarDropDownButton ShowDropDownGlyph="False" Padding="4,0">
    <zm:ToolBarDropDownButton.Icon>
      <zm:BitmapIcon Width="24"
                     Height="24"
                     Source="appMainIcon.ico"
                     Stretch="Fill"
                     StretchDirection="Both" />
    </zm:ToolBarDropDownButton.Icon>
  
    <zm:ToolBarDropDownButton.PopupControl>
      <zm:PopupMenu>
        <zm:MenuItem Header="Exit" />
      </zm:PopupMenu>
    </zm:ToolBarDropDownButton.PopupControl>
  </zm:ToolBarDropDownButton>
</zm:AppWindow.TitleBarHeadContent>
```
For example above to hide a default window icon you can use the `ShowIcon` property. In this case you can specify an icon visible in the task bar, and hidden in the window title bar.

**Comment by JellosPorsche** [comment]
Awesome, thanks for adding this.
